INSTALL-RHEL.md

```diff
@@ -0,0 +1,41 @@
+### Installation on RHEL
+
+The following are instructions for setting up a system starting from a stock system images.
+
+These instructions were tested on a 64-bit systems from https://github.com/opscode/bento, and using the _Junos PyEZ_ library version 2.6.3.
+
+Operating Systems
+---------------
+- Red hat Enterprise Linux 8.1 (Ootpa)
+
+#### Step 1: Install Python and PIP
+
+##### For Python 3.x:
+###### For RHEL:
+
+#### Step 1: Install Junos PyEZ
```

Comment:
It should be step 2, right ?

Reply:
thanks, updated 
#### Step 2: Install Junos PyEZ